gradle/dependencies.gradle

```diff
@@ -70,6 +70,7 @@ def build = [
                                "com.ibm.wala:com.ibm.wala.shrike:${versions.wala}",
                                "com.ibm.wala:com.ibm.wala.core:${versions.wala}"],
     commonscli              : "commons-cli:commons-cli:${versions.commonscli}",
+    jsonSimple              : "com.googlecode.json-simple:json-simple:1.1.1",
```

Comment:
Why do we need this dependence?  From the description I thought this code is outputting `.csv` files?  In general we should only add new dependencies for NullAway if it's really necessary.

Reply:
It was needed for ```Serialization Configuration``` and now is removed as suggested 4eb83ddfdcb282861cb9c624a258c5f6e1e0fd2f